index.js

```diff
@@ -44,17 +44,17 @@ var fs = require('fs'),
   open = require('opn'),
   _ = require('underscore'),
   docopt = require('docopt').docopt,
-  btoa = require('btoa');
+  btoa = require('btoa'),
+  os = require('os');
 
 function computeSpans(mapConsumer, generatedJs) {
-  var lines = generatedJs.split('\n');
+  var lines = generatedJs.split(os.EOL);
```

Comment:
The JS file's line endings might not match the host operating system's. Allowing either line break seems fine, though, so how about replacing this with a regex?

Reply:
How about use regex in [this answer](https://stackoverflow.com/a/21895354)?